traffic_ops/traffic_ops_golang/login/login.go

```diff
@@ -21,8 +21,12 @@ package login
 
 import (
 	"encoding/json"
+	"errors"
 	"fmt"
+	"github.com/dgrijalva/jwt-go"
+	"github.com/lestrrat/go-jwx/jwk"
```

Comment:
These libraries have more easily used parsing and validating functions for jwts and for fetching the keys from the json key set URL. do they not seem good to use?

Reply:
The libraries seem fine, it's just that I don't like pulling in more external dependencies if the standard library would work just as well. I'm especially hesitant about `jwk`, because that doesn't really appear to have any docs.

But after digging through the standard library's Oath2 package ( and its sub-packages ) I wasn't really able to find a way to simply parse a JWT - which is sort of exactly what we want to do here. 

Maybe I'm just an idiot and it can be done, but since I couldn't figure out how I'm fine with just using these I guess. It'd be nice if we could just pick one or the other though (preferably `jwt-go`).

Reply:
Yeah, historically, new project dependencies are something that should really be discussed on the mailing list, primarily because we can get into hot water if we bring in a project dependency that is against the ASF licensing policy. These are both MIT-licensed, which means we should be able to bring them in, but discussing it on the mailing list also lets people evaluate the proposed libraries and give their input on whether or not we should really include them (or if they know of better alternatives -- stuff like that).

Reply:
good to know! I'll send out an email to the mailing list.
And I couldnt figure it out either.  I'll dig more into jwt-go and see if i can get rid of the jwk dependency